react/features/invite/components/add-people-dialog/web/DialInLimit.tsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable lines-around-comment */
+import { Theme } from '@mui/material';
+import React from 'react';
+import { WithTranslation } from 'react-i18next';
+import { makeStyles } from 'tss-react/mui';
+
+import { translate } from '../../../../base/i18n/functions';
+
+const useStyles = makeStyles()((theme: Theme) => {
+    return {
+        limitContainer: {
+            backgroundColor: theme.palette.warning01,
+            borderRadius: '6px',
+            padding: '8px 16px'
+        },
+        limitInfo: {
+            color: theme.palette.field01,
+            fontFamily: 'Inter',
+            fontStyle: 'normal',
+            fontWeight: 'normal',
+            fontSize: '14px',
+            lineHeight: '20px'
```

Comment:
use theme token:  `theme.typography.bodyShortRegular` : check examples in code, e.g.: `...withPixelLineHeight(theme.typography.bodyShortRegular)`